matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -5837,14 +5837,14 @@ Library {
 	}
 	Block {
 	  BlockType		  S-Function
-	  Name			  "Yarp Read"
+	  Name			  "YarpRead"
```

Comment:
Even if WB-Toolbox  will always ship the manually written `.slx`/`.mdl`, I think this space removal is good for eventual consistency with blockfactory libraries generated by https://github.com/robotology/blockfactory/issues/10 .

Reply:
For sure less error prone, it was a trivial change, just some `sed` magic. Lucky us that we use mdl and not slx.

Reply:
OT: To be honest I am still confused on the role of `mdl` and `slx` in WBToolbox. The `.slx` model include `.mdl` one, but has support for the images? We could use the `mdl` also in the example of BlockFactory?

Reply:
I should try again substituting the `slx` with the `mdl` also in the installed tree.

Reply:
Ok I just tried. The models work fine, but the library does not show up in the Simulink Library.

![screenshot](https://user-images.githubusercontent.com/469199/52063542-285a8b00-2573-11e9-9860-49a7b19aa37b.jpeg)

Reply:
I mean, if we really want to avoid having a binary file in our tree we can still install the mdl, and then the user will resave the slx the first time he / she opens the simulink library.

Reply:
Some while ago I stored some information on the export process here (a bit unusual but there was no website at the moment):

https://github.com/robotology/wb-toolbox/blob/368119e32955e8eed5626ddc871bba76f4cd28c1/matlab/CMakeLists.txt#L9-L14

> why we do not run the export_library.m script as part of the build project to create the .slx in the build directory, and then we install as any build artefact?

Usually the `mdl` is exported with the matlab version of the maintainer of the library (:hand:). If users have an old version of Matlab they cannot convert a model / library created with a newer version.

Reply:
Cool. I finally got it, I think. So if we did not wanted to commit a `.slx` we could first save the mantainer's `.mdl` to a `mdl` compatible with 2014b: 
~~~
save_system(libraryName, 'WBToolboxLibrary', 'ExportToVersion', 'R2014B_MDL');
~~~
and then convert it to `R2014B_SLX` just during the build on the user's machine, before its install, to support correctly the `EnableLBRepository` option.

Reply:
Exactly